src/vue-resource-case-converter.js

```diff
@@ -46,42 +54,50 @@ function convertObjectKeys(obj, keyConversionFun) {
   }, Array.isArray(obj) ? [] : {}); // preserve "arrayness"
 }
 
+function mergeOptions(globalOptions, localOptions) {
+  // Combines the value of requestUrlFilter/responseUrlFilter with the global options
+  if (typeof(localOptions) === 'boolean') {
+    return Object.assign({}, globalOptions, {convert: localOptions});
```

Comment:
Sounds good! Would you mind terribly adding that portion? I don't easily have an environment set up to play around with gulp and make sure any changes I made to your pipeline were working.